geomstats/backend/tensorflow/__init__.py

```diff
@@ -252,9 +252,9 @@ def einsum(equation, *inputs, **kwargs):
     input_str_list = input_str.split(',')
 
     is_ellipsis = [input_str[:3] == '...' for input_str in input_str_list]
-    at_least_one_ellipsis = bool(_np.sum(is_ellipsis))
+    all_ellipsis = bool(_np.prod(is_ellipsis))
 
-    if at_least_one_ellipsis:
+    if all_ellipsis:
```

Comment:
Why is it different between pytorch (at_least_one_ellipsis) and tf (all_ellipsis)?

Reply:
Good catch! I had tried something different and forgot to change back.